src/test/store/js-tracer.test.js

```diff
@@ -0,0 +1,170 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import { storeWithProfile } from '../fixtures/stores';
+import { selectedThreadSelectors } from '../../reducers/profile-view';
+import { ensureExists } from '../../utils/flow';
+import { changeShowJsTracerSummary } from '../../actions/profile-view';
+import {
+  getProfileFromTextSamples,
+  getProfileWithJsTracerEvents,
+  type TestDefinedJsTracerEvent,
+} from '../fixtures/profiles/make-profile';
+
+describe('selectors/getJsTracerTiming', function() {
+  describe('full stack-based view', function() {
+    it('has no JS tracer timing if no js tracer info is present', function() {
+      const { profile } = getProfileFromTextSamples('A');
+      const { getState } = storeWithProfile(profile);
+      expect(profile.threads[0].jsTracer).toBe(undefined);
```

Comment:
wondering if this expectation is at the right place in this test

Reply:
I think it is? It's testing both that the jsTracer doesn't exist and the timing doesn't exist.

Reply:
I'm asking because we're in the tests for `selectors/getJsTracerTiming` but this specific expectation is more about the processing process.

The next expectation is good, this comment was solely about the first expectation in this test.

Reply:
Hmm... I think it is where I would expect it, since it's checking the null case for `getJsTracerTiming`.

Reply:
I feel like there's a misunderstanding here :)
The line that checks that `getJsTracerTiming` returns null is totally fine.
What I'm commenting about is this only line 20, checking for `profile.threads[0].jsTracer`.

But if you want to keep it, this is not a big deal. I just want that